cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CruiseControlReconcilerTest.java

```diff
@@ -78,14 +80,16 @@ public void reconcileEnabledCruiseControl(VertxTestContext context) {
         ServiceOperator mockServiceOps = supplier.serviceOperations;
         NetworkPolicyOperator mockNetPolicyOps = supplier.networkPolicyOperator;
         ConfigMapOperator mockCmOps = supplier.configMapOperations;
-
+        PasswordGenerator mockPasswordGenerator = mock(PasswordGenerator.class);
```

Comment:
Why not just do `new PasswordGenerator(10, "a", "a")`?

Reply:
Makes sense.